tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -970,58 +973,96 @@ public String toString() {
   }
 
   private String toStringInternal() {
-    init();
     StringBuilder sb = new StringBuilder();
     if (tableInfo != null) {
       sb.append(String.format("[table: %s] ", tableInfo.getName()));
     }
 
-    boolean isIndexScan = false;
     switch (getIndexScanType()) {
```

Comment:
We also need to change the method `getIndexScanType()` and the  `IndexScanType` to the correct name

Reply:
changed name